src/omlt/block.py

```diff
@@ -241,39 +71,10 @@ def build_formulation(self, formulation, input_vars=None, output_vars=None):
         ----------
         formulation : instance of PyomoFormulation
             see, for example, FullSpaceContinuousFormulation
```

Comment:
this should refer to: `NeuralNetworkFormulation`

Reply:
fixed.